source/digits_hits/src/GateDoseActor.cc

```diff
@@ -360,13 +360,16 @@ void GateDoseActor::UserSteppingActionInVoxel(const int index, const G4Step* ste
   if (mIsDoseImageEnabled) {
     // ------------------------------------
     // Convert deposited energy into Gray
-    dose = edep/density/mDoseImage.GetVoxelVolume()/gray;
+    dose = edep/density/mDoseImage.GetVoxelVolume();
     // ------------------------------------
 
     GateDebugMessage("Actor", 2,  "GateDoseActor -- UserSteppingActionInVoxel:\tdose = "
 		     << G4BestUnit(dose, "Dose")
```

Comment:
Ok. It is the same. But previous version was wrong because conversion was made twice.